docs/source/models/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -311,6 +311,11 @@ category separated by high and low burden locations. This data is stored at :cod
 The trials included in this meta-analysis were identical to those included in the prior meta-analysis. 
 The new data here was the prevalence ratios separated by high and low burden regions. 
 
+In generating the wasting transition data from the prevalence ratios, we encountered a few limitations: 
+
+- The upper bound of the prevalence ratio for low burden regions was greater than 1. This would imply that SQ-LNS has a negative impact. For our model, we set the upper limit to be exactly 1, indicating no impact. 
```

Comment:
```suggestion
- The upper bound of the prevalence ratio for low burden regions was greater than 1. This would imply that SQ-LNS has a negative impact. For our model, we set the upper limit to be exactly 1, indicating no impact. This is implemented in cell [2] of the effect generation notebook, in the line `validation_targets_ucl_low_burden = [1.0, 1.0, 1.0, 1.01]`.
```